src/Libraries/RevitNodes/Elements/FamilyType.cs

```diff
@@ -252,4 +419,23 @@ public override string ToString()
         #endregion
 
     }
+
+    public class FamOpt1 : IFamilyLoadOptions
+    {
+        public FamOpt1() { }
+
+        public bool OnFamilyFound(bool familyInUse, out bool overwriteParameterValues)
+        {
+            overwriteParameterValues = true;
+            return true;
+        }
+
+        public bool OnSharedFamilyFound(Autodesk.Revit.DB.Family sharedFamily, bool familyInUse, out FamilySource source, out bool overwriteParameterValues)
+        {
+            source = FamilySource.Project;
```

Comment:
If this is the case, shall an warning/error message be displayed so that users know that the geometries they passed in is not used at all.